### DISCUSSION TITLE
add polygonal circle creation tool in osu! editor

### DISCUSSION THREAD

**Discussion by nanashi-1** [ppy/osu#19970] [Ideas] [answered]
In osu!stable, this feature generates hit circles on every edge of the set polygon. 

It is located in the menu bar of the osu!stable editor.

![Polygonal Circle Creation Tool](https://user-images.githubusercontent.com/54083498/186778600-34bbe531-55f1-43d2-a1ab-fa9700b963d7.png)

**Comment by Trung0246** [comment]
You can wait for this: https://github.com/ppy/osu/pull/26303

**Comment by bdach** [comment, marked as answer]
https://github.com/ppy/osu/pull/28538
